config/lib/addControllers.js

```diff
@@ -1,43 +1,44 @@
-const express            = require('express');
-const requireDir         = require('require-dir');
-const controllerLocation = process.cwd() + '/controllers';
+"use strict";
 
-/*
+const express = require("express");
+const requireDir = require("require-dir");
+const controllerLocation = process.cwd() + "/controllers";
 
+/*
  The following function adds every file in the directory controllerLocation
  and all its subdirectories as a controller. It excludes any file with
  the name '404', and instead adds the 404 file located in the controllerLocation,
  after all other controllers.
  */
 module.exports = function (app) {
-	let module = {};
+	const module = {};
 
 	module.addControllers = function () {
-		isDirectory = function (file) {
-			return (typeof file !== 'function');
+		const isDirectory = function (file) {
+			return (typeof file !== "function");
 		};
 
-		recursiveAdd = function (dir) {
-			const controllers = requireDir(dir, {recurse: true});
-			for (let i in controllers) {
-				if (i !== '404') {
+		const recursiveAdd = function (dir) {
+			const controllers = requireDir(dir, { recurse: true });
+			for (const i in controllers) {
```

Comment:
why is this became a constant ?

Reply:
ES6 standards denote that you should not use `var` anymore. Either `const` for assigning something to an identifier that will never change again. Or use `let` for identifiers that you will change later on. If I change this to anything other than `const` ESlint will actually start throwing errors.